mono/profiler/mono-profiler-log.c

```diff
@@ -5159,6 +5160,7 @@ mono_profiler_startup (const char *desc)
 		}
 		if ((opt = match_option (p, "heapshot", &val)) != p) {
 			events &= ~MONO_PROFILE_ALLOCATIONS;
+			events &= ~MONO_PROFILE_GC_MOVES;
```

Comment:
Couple of things. I believe the commit message is wrong.. The serve no purpose when allocation profiling is *disabled*.

More importantly, have you discussed this change with the profiler team?